packages/sdk-components-react-remix/src/remix-form.tsx

```diff
@@ -12,13 +18,19 @@ export const RemixForm = forwardRef<
       method?: Lowercase<NonNullable<FormProps["method"]>>;
     }
 >((props, ref) => {
+  const { renderer } = useContext(ReactSdkContext);
   // remix casts action to relative url
   if (
     props.action === undefined ||
     props.action === "" ||
     props.action?.startsWith("/")
   ) {
-    return <Form {...props} ref={ref} />;
+    // remix forms specifies own action instead of provided one
```

Comment:
Is the form supposed to work on preview, like fully functional? If yes how would it work without remix?

Reply:
We do interception to simulate what remix does. But remix does it with all custom workarounds to support server which does not work with our interceptor.